src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -703,6 +703,54 @@ inline void rpp_store48_f32pln3_to_u8pln3_avx(Rpp8u *dstPtrR, Rpp8u *dstPtrG, Rp
     _mm_storeu_si128((__m128i *)dstPtrB, px[2]);    /* store [B01|B02|B03|B04|B05|B06|B07|B08|B09|B10|B11|B12|B13|B14|B15|B16] */
 }
 
+inline void rpp_store48_f32pln3_to_f32pln3_avx(Rpp32f *dstPtrR, Rpp32f *dstPtrG, Rpp32f *dstPtrB, __m256 *p)
+{
+    _mm256_storeu_ps(dstPtrR, p[0]);
+    _mm256_storeu_ps(dstPtrG, p[2]);
+    _mm256_storeu_ps(dstPtrB, p[4]);
+    _mm256_storeu_ps(dstPtrR + 8, p[1]);
+    _mm256_storeu_ps(dstPtrG + 8, p[3]);
+    _mm256_storeu_ps(dstPtrB + 8, p[5]);
+}
+
+inline void rpp_store48_f32pln3_to_f32pkd3_avx(Rpp32f *dstPtr, __m256 *p)
```

Comment:
Also, could you just call the rpp_store24_f32pln3_to_f32pkd3_avx() two times to process all 48

Reply:
We cannot use it because since the destination R G B AVX registers are not stored in continuous indices
In p[6]
p[0], p[2], p[4] - R,G,B registers for 0 - 7 locations
p[1], p[3], p[5] - R,G,B registers for 8 - 15 locations

Reply:
@sampath1117 Could we do the shuffle when its in u8?